docs/pages/setup/reference/license.mdx

```diff
@@ -3,9 +3,13 @@ title: Enterprise License File
 description: Teleport enterprise license file configuration parameters and requirements
 ---
 
-Commercial Teleport subscriptions require a valid license. The license file can
+Commercial self-hosted Teleport subscriptions require a valid license. The license file can
 be downloaded from the [Teleport Customer
-Portal](https://dashboard.gravitational.com/web/login).
+Portal](https://dashboard.gravitational.com/web/login). When downloading the license file the licensed products for the Teleport Customer will list within the download screen. 
+
+<Admonition type="note" title="Not applicable to Teleport Cloud">
```

Comment:
I think we should make this a `Notice` component with `scope={["cloud"}` so only Cloud customers see this.

Reply:
I would leave it in since this a common confusion.  They may also be researching their setup and the enterprise section applies to cloud and enterprise users.

Reply:
That makes sense—what would you think of a `Notice` without the `scope` and with `type="tip`"? I think that would be cleaner than the current Admonition, since the title and body text mainly repeat one another.